zib_uploader/fetch_data.py

```diff
@@ -0,0 +1,71 @@
+from collections import namedtuple
+from typing import Iterator
+
+import requests
+from tqdm import tqdm
+
+ATC_MAPPINGS_URL = 'http://data.bioontology.org/ontologies/ATC/mappings'
+BIOPORTAL_API_KEY = '8b5b7825-538d-40e0-9e9e-5ab9274a9aeb'
```

Comment:
I am assuming here that we are ok with storing the bioportal api key in github. We were already doing that in notebooks, so I guess this is also fine. I don't know where you got it from @dsmits and how sensitive it is.

Reply:
check